src/test/java/com/mongodb/jdbc/MongoSQLMock.java

```diff
@@ -266,6 +288,10 @@ BsonDocument generateRow() {
         array.add(new BsonInt32(3));
         foo.put(ARRAY_COL_LABEL, array);
 
+        BsonDocument fooSubDoc = new BsonDocument();
+        fooSubDoc.put(INT_COL_LABEL, new BsonInt32(5));
```

Comment:
Would it be possible to add a BsonValue that has type information such as objectId, that would help confirm that the toString() is writing the extended json value.

Reply:
now that we decided this ticket will cover stringifying all types as extended json, going to update the unit tests to cover all types

Reply:
done

Reply:
@nbagnard I'm not sure I understand what your suggestion is here. What specific test methods are you suggesting I update/remove/addd? Can you provide an example of what you're envisioning?

Reply:
done. the code turned out slightly longer in each case: 96 lines (new) vs. 95 (old) for schema, and 83 lines (new) vs 36 (old) for data, but we got to cut the comment out from the schema method and I think it should be more readable now. lmk what you think!

Reply:
Yea, maybe part of the JDBC Test Infrastructure epic could be cleaning up some of the clutter we introduced to the unit tests over the last few months 👍